[cutlass backend][BE] Fix two small things in cutlass backend standalone debugger

henrylhtsang opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #148493

Differential Revision: [D70583777](https://our.internmc.facebook.com/intern/diff/D70583777/)

Two really small things: 
* The bits in BlockFillRandomUniform would round float to ints
* when bias exists, the order of args are C, A, B, D

cc @voznesenskym @penguinwu @EikanWang @jgong5 @Guobing-Chen @XiaobingSuper @zhuhaozhe @blzheng @wenzhe-nrv @jiayisunx @ipiszy @yf225 @chenyang78 @kadeng @muchulee8 @amjames @chauhang @aakhundov

henrylhtsang committed:
[cutlass backend][BE] Fix two small things in cutlass backend standalone debugger



Differential Revision: [D70583777](https://our.internmc.facebook.com/intern/diff/D70583777/)

[ghstack-poisoned]

chenyang78 commented on torch/_inductor/codegen/cuda/gemm_template.py:
```diff
@@ -1310,6 +1310,9 @@ def _update_arg_names_for_test_call_statement(
             del arg_names[2]
+        else:
+            # Reorder them as Bias, A, B
+            arg_names[0:3] = [arg_names[2], arg_names[0], arg_names[1]]
```
Being curious, why do we need reordering? Where do we introduce a different order of passing args? Thanks!

henrylhtsang replied:
> Being curious, why do we need reordering? Where do we introduce a different order of passing args? Thanks!

First off, this change only affect standalone debugger and addmm. The reason is that in the main code, when we are generating the cuda kernel, we pass in a input_reorder https://github.com/pytorch/pytorch/blob/main/torch/_inductor/kernel/mm.py#L614

On that note, let me see if I can reuse it

henrylhtsang committed:
Update on "[cutlass backend][BE] Fix two small things in cutlass backend standalone debugger"


Differential Revision: [D70583777](https://our.internmc.facebook.com/intern/diff/D70583777/)

Two really small things: 
* The bits in BlockFillRandomUniform would round float to ints
* when bias exists, the order of args are C, A, B, D

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

henrylhtsang committed:
Update on "[cutlass backend][BE] Fix two small things in cutlass backend standalone debugger"


Differential Revision: [D70583777](https://our.internmc.facebook.com/intern/diff/D70583777/)

Two really small things: 
* The bits in BlockFillRandomUniform would round float to ints
* when bias exists, the order of args are C, A, B, D

cc voznesenskym penguinwu EikanWang jgong5 Guobing-Chen XiaobingSuper zhuhaozhe blzheng wenzhe-nrv jiayisunx ipiszy yf225 chenyang78 kadeng muchulee8 amjames chauhang aakhundov

[ghstack-poisoned]

henrylhtsang commented:
@pytorchbot merge
